components/global-navigation-bar/dropdown-trigger.jsx

```diff
@@ -179,17 +179,20 @@ class GlobalNavigationDropdownTrigger extends React.Component {
 				style={listItemstyle}
 			>
 				{/* eslint-enable jsx-a11y/no-static-element-interactions */}
```

Comment:
This is disabled above, and then re-enabled here.  Can we try to eliminate both of these while we're in this code and updating it?

Reply:
Turns out those weren't needed in the first place! Removed them and ran into 0 linting errors without any additional work.